gef.py

```diff
@@ -7849,8 +7849,9 @@ def _hexdump(self, start_addr, length, arrange_as, offset=0):
             sym = "<{:s}+{:04x}> ".format(*sym) if sym else ""
             mem = read_memory(cur_addr, l)
             val = struct.unpack(fmt_pack, mem)[0]
+            text = "".join([chr(b) if 0x20 <= b < 0x7F else "." for b in mem])
```

Comment:
This will not work as intended on GDB + Python 2

Reply:
Why not? I took it from the hexdump command.